tests/cli/functional/actions/test_select_client_account.py

```diff
@@ -93,40 +98,54 @@ def test_select_client_account_ambiguous_source(mock_click_prompt, test_emitter,
 
 @pytest.mark.parametrize('selection', range(NUMBER_OF_ETH_TEST_ACCOUNTS))
 def test_select_client_account_valid_sources(mocker,
-                                             mock_click_prompt,
+                                             mock_stdin,
                                              test_emitter,
                                              mock_testerchain,
                                              patch_keystore,
                                              mock_accounts,
-                                             selection):
-
-    # Setup
-    mock_click_prompt.return_value = selection
+                                             selection,
+                                             capsys):
 
     # From External Signer
+    mock_stdin.line(str(selection))
     mock_signer = mocker.patch.object(KeystoreSigner, 'from_signer_uri')
     selected_account = select_client_account(emitter=test_emitter, signer_uri=MOCK_SIGNER_URI)
     expected_account = mock_testerchain.client.accounts[selection]
     assert selected_account == expected_account
     mock_signer.assert_called_once_with(uri=MOCK_SIGNER_URI)
+    assert mock_stdin.empty()
+    captured = capsys.readouterr()
+    assert "Select index of account" in captured.out and f"Selected {selection}" in captured.out
```

Comment:
`GENERIC_SELECT_ACCOUNT` in `cli.literature`?

Some other spots as well lower down.

Reply:
Fixed.